tools/twix/src/panels/map/layers/ball_position.rs

```diff
@@ -26,9 +30,11 @@ impl Layer<Field> for BallPosition {
         );
         let ball_position = nao
             .subscribe_buffered_value("Control.main_outputs.ball_position", Duration::from_secs(2));
+        let world_state = nao.subscribe_value("Control.main_outputs.world_state");
```

Comment:
Why do you subscribe the whole worldstate here instead of using the teamball provided by role assigment?
The `ball_in_field` inside the world state is equal to the own ball if there is one and only if there isn't, it uses the actual team ball.

Reply:
Done